Majasdarbs/03-majasdarbs/Gvido-CV-CSS/gvidozarins.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gvido Zarins | FE developer</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div class="nav-bar">
+            <a href="#work-exp">Work experience</a>
```

Comment:
check you navigation. Browsers have auto scroll to the id, so you might consider adding similar ids to the sections, where you wanna scroll to.